account_reconcile_rule_early_payment_discount/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Cyril Gaudin (Camptocamp)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Early Payment Discount Reconciliation Rules',
+    'version': '12.0.1.0.0',
+    'author': 'Camptocamp, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'category': 'Accounting & Finance',
+    'depends': [
+        'account_reconcile_rule',
+        'account_early_payment_discount',
+    ],
+    'website': 'https://github.com/OCA/account-reconcile',
+    'data': [
+        'views/account_reconcile_rule_view.xml',
+    ],
+    'installable': True,
+    'auto_install': True,
```

Comment:
Why would you auto install this?

Reply:
AFAIR this module makes the necessary glue inbetweeen the account reconciliation and the early payment discount. And if you endup with these two modules/functionalities installed, you want to have this one sliently installed.